Cheetah/c/_namemapper.c

```diff
@@ -231,6 +263,19 @@ static PyObject *PyNamemapper_valueForName(PyObject *obj, char *nameChunks[], in
 /* Now the wrapper functions to export into the Python module */
 /* *************************************************************************** */
 
+static PyObject* namemapper_set_complain_func(PYARGS) {
```

Comment:
a link to your source material would be useful https://docs.python.org/2/extending/extending.html#calling-python-functions-from-c